docs/development.md

```diff
@@ -10,7 +10,14 @@ This doc explains how to set up a development environment for Numaflow.
 1. [`protoc`](https://github.com/protocolbuffers/protobuf) for compiling protocol buffers
 1. [`Node.js®`](https://nodejs.org/en/) for running the UI
 1. [`yarn`](https://classic.yarnpkg.com/en/)
-1. [`k3d`](https://k3d.io/) for local development, if needed
+1. Local Kubernetes System - 
+you need one of the following options as your local Kubernetes system for development usage : 
+    1. [`k3d`](https://k3d.io/) 
+    1. [`minikube`](https://minikube.sigs.k8s.io/docs/start/)
+1. Local Container System
+you need one of the following options as your local Containr system to load images for development usage : 
```

Comment:
Can you remove this part? When you install a local k8s cluster, you already had to pick a virtual machine manager, which includes container management tool.